rllib/algorithms/appo/tf/appo_tf_learner.py

```diff
@@ -20,10 +20,10 @@
 _, tf, _ = try_import_tf()
 
 
-class APPOTfLearner(AppoLearner, ImpalaTfLearner):
-    """Implements APPO loss / update logic on top of ImpalaTfLearner."""
+class APPOTfLearner(APPOLearner, IMPALATfLearner):
+    """Implements APPO loss / update logic on top of IMPALATfLearner."""
 
-    @override(ImpalaTfLearner)
+    @override(IMPALATfLearner)
```

Comment:
Do we still need the TF version of IMPALA/APPO?

Reply:
I hear you. :) Would like to make a clean cut soon, but DreamerV3 still only runs on Tf, so we'll need the TfLearner base-class anyways for now.